src/feature/flow_chart_panel/views/SettingsModal.tsx

```diff
@@ -0,0 +1,120 @@
+import ModalCloseSvg from "@src/utils/ModalCloseSvg";
```

Comment:
Do we have a UT for this component? we should write UT's side by side for newly introduced components.